src/pepr/operator/index.ts

```diff
@@ -103,6 +103,14 @@ When(UDSPackage)
     UDSConfig.isIdentityDeployed = false;
   });
 
+// See https://istio.io/latest/docs/ambient/usage/waypoint/#useawaypoint
+When(a.Namespace)
+  .IsCreatedOrUpdated()
+  .WithName("keycloak")
+  .Mutate(req => {
+    req.SetLabel("istio.io/use-waypoint", "waypoint");
+  });
+
```

Comment:
Would it be better to label the keycloak service(s) and/or pod(s) ([ref](https://istio.io/latest/docs/ambient/usage/waypoint/#configure-a-service-to-use-a-specific-waypoint)) which we have control of in the helm chart?

Reply:
Annotating the Service seems to do the trick. I switched to it.